testutils/common.py

```diff
@@ -329,18 +329,19 @@ def wait_for_traefik(gateway_host, routers=[]):
         return
     if routers == []:
         rnames = [
-            "deployments@docker",
-            "deploymentsMgmt@docker",
-            "minio@docker",
-            "deviceauth@docker",
-            "deviceauthMgmt@docker",
-            "inventoryMgmt@docker",
-            "inventoryMgmtV1@docker",
-            "useradm@docker",
-            "useradmNoAuth@docker",
-            "deviceauth@docker",
-            "deviceauthMgmt@docker",
-            "inventoryV1@docker",
+            "deployments@file",
```

Comment:
killing my most precious helpers, huh? :)

so do I understand correctly that we don't need it because the file provider will register the routes immediately when the containers are healthy? - hence - waiting for 'healthy' is now our primary concern?

if yes, I'm ok with this.

it's just important to know if indeed there is no window where everything is healthy and routes are not fully registered yet. otherwise - it's a nightmare to debug, so please make sure!

Reply:
...ok, so I misunderstood, but indeed wait_for_traefik is not used anymore.

so the question of the possibly small time window when routes are not registered still stands. sorry for the confusion :)

Reply:
> killing my most precious helpers, huh? :)

Sorry! :see_no_evil: 

But seriously though, I figured that the tests were initially failing because of another setup race condition where the `backend-test-runner` would start before the entire backend was up, so I created this health-check function to wait for all services to become healthy. However, as I did not find a robust way to check if the entire docker-composition has been created I ended up modifying the `run` and entrypoint for the test-runner itself after already replacing the `wait_for_traefik` with the health check. As the file configuration is loaded immediately after Traefik starts, I was hoping that the `/ping` endpoint would not return successfully until the config is set up, but I did not find any documentation to back this up and your probably right.